app/backend/src/clj/teet/project/project_db.clj

```diff
@@ -97,6 +97,9 @@
           [:activity/_meetings 0
            :db/id]))
 
+(defn meeting-title [db meeting-id]
+  (:meeting/title (du/entity db meeting-id)))
```

Comment:
this should include the meeting number as well, as many meetings can have the same title